README.md

```diff
@@ -40,26 +50,67 @@ We plan to support the following key features of LoRaWAN 1.0.2 specification, ho
+> Refer to our [upgrade guide](/Docs/upgrade.md)  if you are upgrading from a previous release.
+
+### Prerequisites
 The following should be completed before proceeding with the LoRaWAN starter kit development or deployment in your environment.
 
 - You must have an Azure subscription. Get an [Azure Free account]('https://azure.microsoft.com/en-us/offers/ms-azr-0044p/') to get started.
-- We are based on Azure IoT Edge so it is important that you understand the concepts and deployment model for Azure IoT Edge. Refer to Azure [IoT Edge documentation]('https://docs.microsoft.com/en-us/azure/iot-edge/') to see how it works.
-- Understand how LoRa and LoRaWAN works. A great primer is available at the [LoRa Alliance website]('https://lora-alliance.org/resource-hub/what-lorawantm').
+- We are based on Azure IoT Edge so it is important that you understand the concepts and deployment model for Azure IoT Edge. Refer to Azure [IoT Edge documentation](https://docs.microsoft.com/en-us/azure/iot-edge/) to see how it works.
+- Understand how LoRa and LoRaWAN works. A great primer is available at the [LoRa Alliance website](https://lora-alliance.org/resource-hub/what-lorawantm).
 - To test the solution on a device, you need to have a LoRaWAN Device Kit Gateway and a LoRa end node. We have some recommendations in the [Tested Gateways](#tested-gateways) section below.
 
-## Getting Started
+### Deploy to Azure
+ Press on the button here below to start your Azure Deployment.
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fiotedge-lorawan-starterkit%2Fmaster%2FTemplate%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+
+The ARM template will deploy all the required Azure infrastructure to get you started quickly. Specifically, it configures the required Azure Services and  Azure IoT Edge modules. As soon as you connect your IoT Edge device these modules will be pushed onto the gateway and you should have an Azure connected LoRaWAN gateway running. 
+
+- The template will deploy the following resources in your Azure subscription:
+  - [IoT Hub](https://azure.microsoft.com/en-us/services/iot-hub/)
+  - [Azure Functions](https://azure.microsoft.com/en-us/services/functions/)
+  - [Redis Cache](https://azure.microsoft.com/en-us/services/cache/)
+
+- The LoRaWAN gateway will be provisioned with two Azure IoT Edge modules each for the [packet forwarder](https://github.com/Lora-net/packet_forwarder) and a network server.
 
-We have a variety of ways you can  get started with the kit, chose the appropriate documentation based on your persona and applicability.
-- **Setup a LoRaWAN Gateway**: We provide an easy to use Azure ARM template and deployment guidance to get you quickly started with the LoRaWAN starter kit. Use the [Quick Start](/Docs/quickstart.md) to setup a LoRaWAN Gateway and connect to LoRA end nodes. 
```

Comment:
The first bullet point should highlight that this is the Quick getting started.. I would change it to:
Quick Start - Setup a LoRaWAN gateway: ....